airbyte-integrations/connectors/destination-aws-datalake/Dockerfile

```diff
@@ -13,5 +13,5 @@ RUN pip install .
 ENV AIRBYTE_ENTRYPOINT "python /airbyte/integration_code/main.py"
 ENTRYPOINT ["python", "/airbyte/integration_code/main.py"]
 
-LABEL io.airbyte.version=0.1.1
+LABEL io.airbyte.version=0.1.2
```

Comment:
The change introduces a backward compability, please update to 0.2.0

Reply:
This change has been made.